pkg/generators/models/models_test.go

```diff
@@ -15,133 +15,140 @@ import (
-		},
-		{
-			name:      "allOf merges multiple inlined object definitions on property level",
-			directory: "testdata/cases/allof1",
-		},
-		{
-			name:      "allOf merges multiple inlined object definitions on top level",
-			directory: "testdata/cases/allof2",
-		},
-		{
-			name:      "allOf merges refs and inlined object definitions",
-			directory: "testdata/cases/allof3",
-		},
-		{
-			name:      "allOf can be used in conjunction with nullable to produce pointers to other types",
-			directory: "testdata/cases/allof4",
-		},
-		{
-			name:      "allOf can flatten fields and inline as subtypes of a struct",
-			directory: "testdata/cases/allof5",
-		},
-		{
-			name:      "allOf can skip certain mixins",
-			directory: "testdata/cases/allof_mixin_skipping",
-		},
-		{
-			name:      "typed arrays generated typed arrays in go",
-			directory: "testdata/cases/typed_arrays",
-		},
-		{
-			name:      "if the array prop is nullable it is NOT converted to a pointer (slices are already nullable in go)",
-			directory: "testdata/cases/nullable_arrays",
-		},
-		{
-			name:      "if an untyped object prop is nullable it is NOT converted to a pointer (maps are already nullable in go)",
-			directory: "testdata/cases/nullable_untyped_object",
-		},
-		{
-			name:      "an untyped object may have additional properties of a specific type",
-			directory: "testdata/cases/object_with_additional_properties",
-		},
-		{
-			name:      "an untyped toplevel object may have additional properties of a specific type",
-			directory: "testdata/cases/toplevel_object_with_additional_properties",
-		},
-		{
-			name:      "handles string enums",
-			directory: "testdata/cases/enum",
-		},
-		{
-			name:      "enum with one item",
-			directory: "testdata/cases/enum_with_one_item",
-		},
-		{
-			name:      "model from path body",
-			directory: "testdata/cases/model_from_path_body",
-		},
-		{
-			name:      "enum with name collision",
-			directory: "testdata/cases/enum_with_name_collision",
-		},
-		{
-			name:      "model from path parameters",
-			directory: "testdata/cases/parameter_model",
-		},
-		{
-			name:      "model with recursive definition within an allof",
-			directory: "testdata/cases/allof_self_reference",
-		},
-		{
-			name:      "allof merges required list",
-			directory: "testdata/cases/allof_merges_required_list",
-		},
-		{
-			name:      "allof merges enum list",
-			directory: "testdata/cases/allof_enum",
-		},
-		{
-			name:      "allof supports intermediate array types",
-			directory: "testdata/cases/allof_arrays",
-		},
-		{
-			name:      "oneOf supports mapped discriminators",
-			directory: "testdata/cases/oneof_mapped_discriminator",
-		},
-		{
-			name:      "oneOf supports unmapped discriminators",
-			directory: "testdata/cases/oneof_unmapped_discriminator",
-		},
-		{
-			name:      "example of naming for nested inline arrays",
-			directory: "testdata/cases/nested_inline_arrays",
-		},
-	}
+var cases = []struct {
```

Comment:
I pulled these cases up, so that it was easier to run exactly one test without needing to copy and paste as much